src/components/search/SearchItemStats.tsx

```diff
@@ -3,9 +3,11 @@ import { DisplayableBusiness } from '@futureproof/typings'
 import React from 'react'
 import { Image, StyleSheet, Text, View } from 'react-native'
 
-const SearchItemStats = ({business}: {business: DisplayableBusiness}) => (
+type ComponentProps = Pick<DisplayableBusiness, 'name' | 'customerScore' | 'sustainabilityScore'>
+
+const SearchItemStats = ({ name, customerScore, sustainabilityScore } : ComponentProps) => (
   <View style={styles.searchItemStats}>
-    <Text style={styles.businessTitleText}>{business.name}</Text>
+    <Text style={styles.businessTitleText}>{name}</Text>
     <View style = {styles.logoAndSubtitleView}>
       <Image
         source={require('../../../assets/icon_location.png')}
```

Comment:
change image source path to '@images/icon_location.png'

Reply:
See [above](https://github.com/CM20314-Group-2/client/pull/2#discussion_r810643157).